Documentation - load image resource doesn't compile on Windows

### Description

https://docs.microsoft.com/en-us/dotnet/maui/user-interface/graphics/paint#create-an-imagepaint-object
This section shows how to draw an image on a canvas.
Loading the image doesn't work _**(does not compile)**_ on Windows (it only works on Android (I have tried Windows and Android)) and there is not any mention of this in the article and there is not any mention of how to load the image on Windows.
Isn't MAUI supposed to be cross-platform?
Isn't there any nice and clean code to load the image cross-platform?

I can see there that build action "EmbeddedResource" must be used instead of "MauiImage" which is default.
I assume that EmbeddedResource creates something like a byte[] resource while MauiImage creates a whole set of images which have specific resolution for each platform etc.

Do I have to put a duplicate image into the resources just so that the same image can be used as a image in ImageButton and at the same time drawn on a canvas (the first one requires MauiImage and the second one EmbeddedResource)?

Thank you

Edit:
- added more info:

`error CS0234: The type or namespace name 'Platform' does not exist in the namespace 'Microsoft.Maui.Graphics' (are you missing an assembly reference?)
`

![image](https://user-images.githubusercontent.com/64279914/164211514-caa24102-189b-4dc9-83d6-43bb72b8941e.png)

![image](https://user-images.githubusercontent.com/64279914/164211629-ff077f40-51e1-4660-88c5-cbca21e374e1.png)



### Steps to Reproduce

Try the example on Windows

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 10

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

--- a/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/PlatformImage.cs
+++ b/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/PlatformImage.cs
@@ -7,18 +7,18 @@ using Windows.Storage.Streams;
 
 namespace Microsoft.Maui.Graphics.Win2D
 {
-	internal class W2DImage : IImage
+	public class PlatformImage : IImage
 	{
 		private readonly ICanvasResourceCreator _creator;
 		private CanvasBitmap _bitmap;
 
-		public W2DImage(ICanvasResourceCreator creator, CanvasBitmap bitmap)
+		public PlatformImage(ICanvasResourceCreator creator, CanvasBitmap bitmap)
 		{
 			_creator = creator;
 			_bitmap = bitmap;
 		}
 
-		public CanvasBitmap PlatformImage => _bitmap;
+		public CanvasBitmap Bitmap => _bitmap;
 
 		public void Dispose()
 		{
@@ -118,7 +118,7 @@ namespace Microsoft.Maui.Graphics.Win2D
 				throw new Exception("No resource creator has been registered globally or for this thread.");
 
 			var bitmap = AsyncPump.Run(async () => await CanvasBitmap.LoadAsync(creator, stream.AsRandomAccessStream()));
-			return new W2DImage(creator, bitmap);
+			return new PlatformImage(creator, bitmap);
 		}
 	}
 }

--- a/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/W2DCanvas.cs
+++ b/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/W2DCanvas.cs
@@ -374,9 +374,9 @@ After:
 
 			if (paint is ImagePaint imagePaint)
 			{
-				if (imagePaint.Image is W2DImage image)
+				if (imagePaint.Image is PlatformImage platformImage)
 				{
-					var bitmapBrush = new CanvasImageBrush(_session, image.PlatformImage)
+					var bitmapBrush = new CanvasImageBrush(_session, platformImage.Bitmap)
 					{
 						ExtendX = CanvasEdgeBehavior.Wrap,
 						ExtendY = CanvasEdgeBehavior.Wrap
@@ -510,17 +510,17 @@ After:
 
 		public override void DrawImage(IImage image, float x, float y, float width, float height)
 		{
-			if (image is W2DImage platformImage)
+			if (image is PlatformImage platformImage)
 			{
 				SetRect(x, y, width, height);
 
-/* Unmerged change from project 'Microsoft.Maui.Graphics.Win2D.WinUI.Desktop'
-Before:
-				Draw(s => s.DrawImage(platformImage.PlatformImage, _rect, Rect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
-After:
-				Draw(s => s.DrawImage(platformImage.PlatformImage, _rect, global::Windows.Foundation.Rect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
-*/
-				Draw(s => s.DrawImage(platformImage.PlatformImage, _rect, WRect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
+				/* Unmerged change from project 'Microsoft.Maui.Graphics.Win2D.WinUI.Desktop'
+				Before:
+								Draw(s => s.DrawImage(platformImage.Bitmap, _rect, Rect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
+				After:
+								Draw(s => s.DrawImage(platformImage.Bitmap, _rect, global::Windows.Foundation.Rect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
+				*/
+				Draw(s => s.DrawImage(platformImage.Bitmap, _rect, WRect.Empty, CurrentState.Alpha, CanvasImageInterpolation.Linear));
 			}
 		}
 

--- a/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/W2DImageLoadingService.cs
+++ b/src/Microsoft.Maui.Graphics.Win2D.WinUI.Desktop/W2DImageLoadingService.cs
@@ -8,7 +8,7 @@ namespace Microsoft.Maui.Graphics.Win2D
 	{
 		public IImage FromStream(Stream stream, ImageFormat formatHint = ImageFormat.Png)
 		{
-			return W2DImage.FromStream(stream, formatHint);
+			return PlatformImage.FromStream(stream, formatHint);
 		}
 	}
 

